### PR TITLE
Evergreen Post--work in progress

### PR DESCRIPTION
**Pull request by atribolet**
Pull request description
------------------------

### CODE REVIEW

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "don't make users think about updating software"
+title: "Don't Make Users Think about Updating Software"
```
Comment (atribolet) [inline _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: I typically use Chicago's headline-style capitalization, which capitalizes these terms. But this is a stylistic choice and we don't have a standard on it yet, so your call!

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md [hunk 2]
```diff
@@ -11,35 +11,35 @@ tags:
   - update-service
 ---
 
-As software developers, we want the people using the software we create to ~~enjoy~~ not _hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, many hours are spent debating and designing user interactions that are simple, effective, and intuitive.
+As software developers, we want the people who are using our software to ~~enjoy~~ _not hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, we spend many hours designing, debating, and redesigning to ensure that user interactions with our software are simple, effective, and intuitive.
```
Comment (atribolet) [inline _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: These are mostly just changes for readability. If you prefer the original, feel free to change it back.

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md [hunk 3]
```diff
@@ -11,35 +11,35 @@ tags:
...
 
 ### skip to the [tl;dr <i class="fas fa-long-arrow-alt-down"></i>](#tldr)
 
-> The people that are using your software are motivated by the desire to save time
+> The people who are using your software are motivated by the desire to save time.
```
Comment (atribolet) [inline _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: These subheading lines (whatever you want to call them) are complete sentences, so I went ahead and added ending punctuation.

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md [hunk 4]
```diff
@@ -11,35 +11,35 @@ tags:
...
 
-Building on the blog post from Scott Davis, _[Cache breaking for faster web application updates]({{site.baseurl}}{% post_url 2018-03-23-cache-breaking-for-faster-web-application-updates %})_, which is about website caching techniques for delivering faster updates to users, at it's core is about improving the user experience of website applications. Getting updates to users is an often overlooked feature and an after thought. But it is very important—_Especially when you ship a lot of bugs_.
+Building on the blog post from Scott Davis, _[Cache Breaking for Faster Web Application Updates]({{site.baseurl}}{% post_url 2018-03-23-cache-breaking-for-faster-web-application-updates %})_, which is about website caching techniques for delivering faster updates to users, at its core software development is about improving the user experience of website applications. Getting updates to users is an often overlooked feature and an afterthought. But it is very important—_especially when you ship a lot of bugs_.
```
Comment (atribolet) [inline _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: Hmm...might need a comma after "core"...Also, did I understand that correctly? "at its core, software development is about..."?

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md [hunk 5]
```diff
@@ -11,35 +11,35 @@ tags:
...
+Modern browsers and app stores remove the hurdles to keeping your and their software evergreen. The term `evergreen` might have become popular when Google Chrome started auto updating in late 2010. Since then, all major browsers have switched to auto updating. Browser auto updating is pretty straightforward&mdash; when you start your browser, it checks for an update and downloads it in the background. Then, the next time you start the software, you are using the newest version. I think this set a precedent for software that users expect and desire. Going to a website to download an installer to keep someone else's software current should not be a responsibility of the user. And placing this responsibility on the users is a great way to turn them away.
```
Comment (atribolet) [inline _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: Changed that "Browser auto updating is pretty straightforward" for readability again. The "Browser auto updating works by when..." wording threw me off a little bit. 

Also, auto updating: Chicago Manual of Style (which I typically follow for AGRC stuff) usually recommends making these kinds of terms one word, no hyphen. Consistency is most important, but personally, I'd recommend either adding a hyphen or making it one word no hyphen.

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md [hunk 6]
```diff
@@ -49,35 +49,35 @@ We ran into the problem of people using outdated software with deprecated urls a
...
 
 > evergreen.Update(newVersion);
 
-The developer has the choice of how or if they want to notify the user of the update. Updates can be made silently or a notification could be displayed; The UI is up to the add-in developer. Using GitHub release and pre-release assets, users can be put in "beta" or "stable" channels. This allows the developer to test changes before delivering to a larger audience. There are a lot of benefits to this approach and we hope you can benefit from our experience and efforts.
+The developer has the choice of how (and whether) they want to notify the user of the update. Updates can be made silently or a notification could be displayed; the UI is up to the add-in developer. Using GitHub release and pre-release assets, users can be put in "beta" or "stable" channels. This allows the developer to test changes before delivering to a larger audience. There are a lot of benefits to this approach, and we hope you can benefit from our experience and efforts.
```
Comment (atribolet) [inline _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: Kind of sounds weird, but "whether" is used when there are two options: the developer can either (1) notify the user or (2) not notify the user.

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md [hunk 7]
```diff
@@ -49,35 +49,35 @@ We ran into the problem of people using outdated software with deprecated urls a
...
 
 #### Getting Started
 
-The evergreen documentation is in the repository [read me](https://github.com/steveoh/pro-evergreen/blob/master/README.md) but getting started with Pro Evergreen is 10 steps or less.
+The evergreen documentation is in the repository [read me](https://github.com/steveoh/pro-evergreen/blob/master/README.md) and getting started with Pro Evergreen is 10 steps or fewer.
```
Comment (atribolet) [inline _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: Again, a little weird, but "fewer" is used with concrete/countable objects. This topic is a little subjective though, so take that for what you will!

**Comment by atribolet** [comment]
K, finished with the initial edits, @steveoh 

Let me know if you have questions!

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md (hunk 4 above)
Comment (steveoh) [reply _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: yeah i wasn't sure, it seemed like there was a lot of commas in this sentence. I thought you might add some dashes and other stuff! I was really curious to see what you did here...

So the at it's core is referencing scott's posts content not software development in general.

File: _posts/2018-04-11-dont-make-users-think-about-updating-software.md (hunk 5 above)
Comment (steveoh) [reply _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: I'll switch it to one word

Comment (steveoh) [reply _posts/2018-04-11-dont-make-users-think-about-updating-software.md]: image

even though a hyphen is twice as popular

**Pull request merged**